modules/graceful-shutdown.adoc

```diff
@@ -1,102 +1,90 @@
 //
 // * backup_and_restore/graceful-cluster-shutdown.adoc
-
 :_content-type: PROCEDURE
 [id="graceful-shutdown_{context}"]
 = Shutting down the cluster
-
 You can shut down your cluster in a graceful manner so that it can be restarted at a later date.
-
 [NOTE]
 ====
 You can shut down a cluster until a year from the installation date and expect it to restart gracefully. After a year from the installation date, the cluster certificates expire.
 ====
-
 .Prerequisites
-
 * You have access to the cluster as a user with the `cluster-admin` role.
 * You have taken an etcd backup.
-
-.Procedure
-
-. If you plan to shut down the cluster for an extended period of time, determine the date that cluster certificates expire.
 +
-You must restart the cluster prior to the date that certificates expire. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
-
-.. Check the expiration date for the `kube-apiserver-to-kubelet-signer` CA certificate:
+[IMPORTANT]
+====
+It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues when restarting the cluster.
+For example, the following conditions can cause the restarted cluster to malfunction:
+* etcd data corruption during shutdown
+* Node failure due to hardware
+* Network connectivity issues
+If your cluster fails to recover, follow the steps to restore to a previous cluster state.
+====
+.Procedure
+. If you are shutting the cluster down for an extended period, determine the date on which certificates expire.
 +
 [source,terminal]
 ----
-$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}{"\n"}'
+$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
 ----
 +
 .Example output
-[source,terminal]
 ----
-2023-08-05T14:37:50Z
+2022-08-05T14:37:50Zuser@user:~ $ <1>
 ----
-
-.. Check the expiration date for the kubelet certificates:
-
-... Start a debug session for a control plane node by running the following command:
+. Mark all the nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
 +
-[source,terminal]
-----
-$ oc debug node/<node_name>
-----
-
-... Change your root directory to `/host` by running the following command:
-+
-[source,terminal]
+[source,yaml]
 ----
-sh-4.4# chroot /host
-----
-
-... Check the kubelet client certificate expiration date by running the following command:
-+
-[source,terminal]
-----
-sh-5.1# openssl x509 -in /var/lib/kubelet/pki/kubelet-client-current.pem -noout -enddate
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm cordon ${node} ; done
 ----
```

Comment:
Please check the "Example output" of this step, it is not correct.  And the following content like "Check the kubelet server certificate expiration date by running the following command:" is not relevant and it appears twice. Please confirm and delete